Spigot-Server-Patches/0698-forced-whitelist-use-configurable-kick-message.patch

```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Trigary <trigary0@gmail.com>
+Date: Sat, 27 Mar 2021 09:24:23 +0100
+Subject: [PATCH] forced whitelist: use configurable kick message
+
+
+diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
+index c372e6897c73fad6f9c478cec9f58303fb0cce17..77f8be879ba098aaa15a5fd947dc3a4b39c5e397 100644
+--- a/src/main/java/net/minecraft/server/MinecraftServer.java
++++ b/src/main/java/net/minecraft/server/MinecraftServer.java
+@@ -2030,7 +2030,7 @@ public abstract class MinecraftServer extends IAsyncTaskHandlerReentrant<TickTas
+                 EntityPlayer entityplayer = (EntityPlayer) iterator.next();
+ 
+                 if (!whitelist.isWhitelisted(entityplayer.getProfile())) {
+-                    entityplayer.playerConnection.disconnect(new ChatMessage("multiplayer.disconnect.not_whitelisted"));
++                    entityplayer.playerConnection.disconnect(org.spigotmc.SpigotConfig.whitelistMessage);
```

Comment:
// Paper

Reply:
Ty, don't know how I managed to forget that.